### PR TITLE
rework manifest

### PR DESCRIPTION
**Pull request by maggu2810**
Please have a look at the manifest yourself, too -- it has been done by a tool.

Related to: https://github.com/eclipse/smarthome/pull/4308

### CODE REVIEW

File: addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
- org.osgi.service.component,
- org.osgi.service.component.annotations;resolution:=optional,
- org.slf4j,
- org.openhab.binding.allplay,
- org.openhab.binding.allplay.handler
-Service-Component: OSGI-INF/*.xml
-Export-Package: org.openhab.binding.allplay,
- org.openhab.binding.allplay.handler
+Manifest-Version: 1.0
+Bundle-ClassPath: .,lib/tchaikovsky.jar
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF]: I checked the auto-formatting of Eclipse, it changes this line.

File: addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -1,36 +1,40 @@
...
+Bundle-ManifestVersion: 2
+Bundle-Name: AllPlay Binding
+Bundle-NativeCode: lib/x86-64/win/alljoyn_java.dll;processor=amd64;osnam
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF]: Maybe i would prefer breaking on a more natural place if that is allowed

**Comment by martinvw** [comment]
@maggu2810 is your tool available somewhere, maybe as a website so I can request people to apply that formatting to new PR's?

**Comment by maggu2810** [comment]
@martinvw Not yet, will ping you as soon as it is.

File: addons/binding/org.openhab.binding.amazondashbutton/META-INF/MANIFEST.MF [hunk 3]
```diff
@@ -1,35 +1,38 @@
...
+Manifest-Version: 1.0
+Bundle-ClassPath: 
+ .,
+ lib/jna-4.2.1.jar,
+ lib/pcap4j-core-1.6.6.jar,
+ lib/pcap4j-packetfactory-static-1.6.6.jar
+Bundle-ManifestVersion: 2
+Bundle-Name: AmazonDashButton Binding
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.amazondashbutton;singleton:=tru
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.amazondashbutton/META-INF/MANIFEST.MF]: Do you use a fixed length for line breaks? This looks a little bit weird. And I suppose it won't work.

File: addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF [hunk 4]
```diff
@@ -1,36 +1,45 @@
...
-Export-Package: org.openhab.binding.allplay,
- org.openhab.binding.allplay.handler
+Manifest-Version: 1.0
+Bundle-ClassPath: 
+ .,
+ lib/tchaikovsky.jar
+Bundle-ManifestVersion: 2
+Bundle-Name: AllPlay Binding
+Bundle-NativeCode: 
+ lib/x86-64/win/alljoyn_java.dll;processor=amd64;osname=win32;osname=Win
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.allplay/META-INF/MANIFEST.MF]: Fix line breaks.

File: addons/binding/org.openhab.binding.max.test/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -1,12 +1,15 @@
 Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: openHAB Max! Binding Tests
-Bundle-SymbolicName: org.openhab.binding.max.test;singleton:=true
-Bundle-Version: 2.2.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.max.test;singleton:=true
 Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Export-Package: org.openhab.binding.max.test;uses:="org.eclipse.smarthom
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.max.test/META-INF/MANIFEST.MF]: Fix line breaks.

File: addons/binding/org.openhab.binding.nest.test/META-INF/MANIFEST.MF [hunk 6]
```diff
@@ -1,25 +1,29 @@
...
-Export-Package: 
- org.openhab.binding.nest;uses:="org.eclipse.smarthome.test"
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name:  Nest Binding Tests
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-SymbolicName: org.openhab.binding.nest.test;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Export-Package: org.openhab.binding.nest;uses:="org.eclipse.smarthome.te
```
Comment (cweitkamp) [inline addons/binding/org.openhab.binding.nest.test/META-INF/MANIFEST.MF]: Fix line breaks.

**Comment by maggu2810** [comment]
About your line break comments: https://github.com/openhab/openhab-core/pull/211#issuecomment-330878293

**Comment by cweitkamp** [comment]
I see, thanks.

**Comment by martinvw** [comment]
@maggu2810 sorry this PR went a little of my radar and there are some merge conflicts now, if you can fix those and ping me I should be able to merge them quickly.

**Comment by maggu2810** [comment]
ping

**Comment by martinvw** [comment]
Thanks!

**Comment by martinvw** [comment]
@ others sorry for any potential conflicts in your branches...

**Pull request merged**